### PR TITLE
Tests for internal '@meta' and '@removed' tags. Added php7.4 typehints

### PR DESCRIPTION
**Pull request by isfedorov**

### CODE REVIEW

File: tests/Model/BasePHPElement.php [hunk 1]
```diff
@@ -24,8 +27,13 @@ abstract class BasePHPElement
...
 
-    abstract public function readMutedProblems($jsonData);
+    abstract public function readMutedProblems($jsonData): void;
 
+    /**
+     * @param Node $node
+     *
+     * @return string
+     */
     protected function getFQN($node): string
```
Comment (MaXal) [inline tests/Model/BasePHPElement.php]: Let's add the type to the signature and remove PHPDoc

File: tests/Model/BasePHPElement.php [hunk 2]
```diff
@@ -40,6 +48,11 @@ protected function getFQN($node): string
         return rtrim($fqn, "\\");
     }
 
+    /**
+     * @param $stubProblemType
+     *
+     * @return bool
+     */
     public function hasMutedProblem($stubProblemType): bool
```
Comment (MaXal) [inline tests/Model/BasePHPElement.php]: Unnecessary PHPDoc since it contains no additional info

File: tests/Model/PHPDefineConstant.php [hunk 3]
```diff
@@ -1,4 +1,5 @@
 <?php
+/** @noinspection PhpMissingStrictTypesDeclarationInspection */
```
Comment (MaXal) [inline tests/Model/PHPDefineConstant.php]: Let's fix warnings instead of suppressing them.

File: tests/Model/PHPFunction.php [hunk 4]
```diff
@@ -19,17 +18,17 @@ class PHPFunction extends BasePHPElement
     use PHPDocElement;
 
     /**
-     * @var boolean $is_deprecated
+     * @var bool $is_deprecated
```
Comment (MaXal) [inline tests/Model/PHPFunction.php]: Let's remove "empty" PHPDocs

File: tests/Model/PHPClass.php [hunk 5]
```diff
@@ -59,7 +56,7 @@ public function readObjectFromStubNode($node): self
     {
         $this->name = $this->getFQN($node);
         $this->collectLinks($node);
-        $this->collectSinceDeprecatedVersions($node);
+        $this->collectSinceRemovedDeprecatedVersions($node);
```
Comment (MaXal) [inline tests/Model/PHPClass.php]: Let's check `@since` as well for the consistency

Comment (isfedorov) [reply tests/Model/PHPClass.php]: Such test already exists

**Pull request merged**